sources/HUBComponentWrapper.h

```diff
@@ -167,6 +167,9 @@ willUpdateSelectionState:(HUBComponentSelectionState)selectionState;
 /// Whether the wrapped component's view has appeared since the model was last changed
 @property (nonatomic, readonly) BOOL viewHasAppearedSinceLastModelChange;
 
+/// Whether the wrapped component's view has appeared for the first time
+@property (nonatomic, assign, nullable, readonly) NSNumber *viewHasAppearedForTheFirstTime;
```

Comment:
Alternatively, an enum:
```
enum ApperanceState {
    case hasNotAppeared
    case hasAppearedOnce
    case hasAppearedMultipleTimes
}
```

Personally, I think I prefer the integer approach (if possible). Simpler 😄